ros/lib/config.py

```diff
@@ -15,6 +15,7 @@
 INVENTORY_URL = f"http://{INVENTORY_HOST}:{INVENTORY_PORT}"
 INVENTORY_EVENTS_TOPIC = os.getenv("INVENTORY_EVENTS_TOPIC", "platform.inventory.events")
 
+Engine_RESULT_TOPIC = os.getenv("INVENTORY_EVENTS_TOPIC", "platform.engine.results")
```

Comment:
```suggestion
Engine_RESULT_TOPIC = os.getenv("Engine_RESULT_TOPIC", "platform.engine.results")
```

Reply:
Just a minor nitpick if we can make it:
```suggestion
ENGINE_RESULT_TOPIC = os.getenv("ENGINE_RESULT_TOPIC", "platform.engine.results")
```

Reply:
Thanks @apuntamb  for correction :)